src/webgpu/capability_info.ts

```diff
@@ -89,9 +89,10 @@ export const kTextureFormatInfo: {
   'rgba32sint':             { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, copyable:  true, bytesPerBlock: 16, blockWidth: 1, blockHeight: 1 },
   'rgba32float':            { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, copyable:  true, bytesPerBlock: 16, blockWidth: 1, blockHeight: 1 },
   // Depth/stencil formats
-  'depth32float':           { renderable:  true, color: false, depth:  true, stencil: false, storage: false, copyable:  true, bytesPerBlock:  4, blockWidth: 1, blockHeight: 1 },
-  'depth24plus':            { renderable:  true, color: false, depth:  true, stencil: false, storage: false, copyable: false,                                                  },
-  'depth24plus-stencil8':   { renderable:  true, color: false, depth:  true, stencil:  true, storage: false, copyable: false,                                                  },
+  // bytesPerBlock are undefined for depth24plus and depth24plus-stencil8
```

Comment:
The optional values are causing a lot of warnings. Is there a way to suppress them?

Also we can't copy to depth32float, so for now it seems that copyable should be false there.

Reply:
This is a problem that I've been wanting to fix but haven't had a chance. We need to split this texture table between color and depth/stencil formats, so when you only want color formats you can use one that doesn't have the optionals.

You should be able to suppress it by doing something like `kTextureFormatInfo[format].bytesPerBlock!`. The `!` operator just removes the possibility of `undefined` from a type.

Reply:
I removed the `copyable` change as well and now the `formatCopyableWithMethod` helper function deals with depth32float.